.github/quarkus-ecosystem-test

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -e
+
+mkdir -p ~/.m2 ; cp .github/settings.xml ~/.m2/
+
+# update Quarkus dependencies
+# TODO: aligning parent pom versions with upstream Quarkus breaks compilation
```

Comment:
@pedroigor / @DGuhr this is something we probably want to target, running a command like:
```
mvn versions:compare-dependencies \
    -pl :keycloak-parent \
    -DremotePom=io.quarkus:quarkus-bom:${QUARKUS_VERSION} \
    -DupdatePropertyVersions=true \
    -DupdateDependencies=true \
    -DgenerateBackupPoms=false
```

breaks the compilation.